packages/vscode-extension/src/extension.ts

```diff
@@ -30,6 +30,7 @@ export async function activate(context: ExtensionContext) {
     debug: {
       module: debugServer,
       transport: TransportKind.ipc,
+      options: { env: { CYPHER_25: 'true' } },
```

Comment:
This only affects the server version we use in the tests, not the production one. So we'll have `CYPHER_25` support enabled there by default.